packages/form/src/form-item.vue

```diff
@@ -193,16 +191,20 @@ export default defineComponent({
       }
       return required
     })
-    const elFormItemSize = computed(() => props.size || elForm.size)
+    const elFormItemSize = computed(() => props.size || elForm?.size)
     const sizeClass = computed(() => {
       return elFormItemSize.value || $ELEMENT.size
     })
 
     const validate = (trigger: string, callback?: ValidateFieldCallback) => {
+      if (trigger === 'change' && validateDisabled.value) {
+        validateDisabled.value = false
+        return
+      }
       validateDisabled.value = false
       const rules = getFilteredRule(trigger)
       if ((!rules || rules.length === 0) && props.required === undefined) {
-        callback()
+        if (callback) callback()
```

Comment:
Can we change here like `callback?.()`, I think this is the perfect place for us using this syntax.